src/vma/main.cpp

```diff
@@ -406,6 +408,10 @@ void print_vma_global_settings()
 		vlog_printf(VLOG_INFO,"OFED Version: %s\n", ofed_version_info);
 	}
 
+	if (vma_git_version_str[sizeof(VMA_GIT_TITLE)-1]) {
```

Comment:
The purpose of this check is to verify that git version exists (non-zero)
If git version cannot be achieved (compiling in a non-git environment) - banner may not include git information.